db/seeds.rb

```diff
@@ -62,6 +62,18 @@
     'lightdash',
     'multiverse'
   ],
```

Comment:
Can you add workday jobs in due course for JB

Reply:
Yep